dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -837,15 +842,21 @@ LANStatusDlg(
     switch (uMsg)
     {
         case WM_INITDIALOG:
-            pContext = (LANSTATUSUI_CONTEXT *)lParam;
-            SetWindowLongPtr(hwndDlg, DWLP_USER, (LONG_PTR)lParam);
```

Comment:
You no longer associate the context with the window, so now all other functions are broken?

Reply:
You're right. Fixed